headless-Checkout/client/react/headless-checkout/src/pages/make-payment/PaymentMethodOptions.jsx

```diff
@@ -0,0 +1,217 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+export default function PaymentMethodOptions() {
+    const create_order_url = 'http://localhost:5009/v1/orders';
+    let payment_method_options_url = 'http://localhost:5009/v1/payment-method-options';
+
+    const navigate = useNavigate();
+
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(false);
+    const [paymentMethodOptions, setPaymentMethodOptions] = useState([]);
+    const shouldRenderPaymentMethodOptions = !error && !loading && paymentMethodOptions.length;
+    const [selectedPaymentMethod, setSelectedPaymentMethod] = useState(null);
+    const [paymentDetails, setPaymentDetails] = useState({});
+    const [orderId, setOrderId] = useState('');
+
+    const getPaymentMethods = async () => {
+        try{
+            // create order
+            const order_response = await fetch(create_order_url, {
+                method: 'POST',
+                headers: {
+                    Accept: 'application/json', 
+                    'Content-Type': 'application/json',
+                },
+                body : JSON.stringify({
+                    customer: {
+                        external_id: process.env.REACT_APP_EXTERNAL_ID // merchant's representation of a customer
+                    }
+                })
+            });
+            const order_response_data = await order_response.json();
+            if (order_response.status !== 201) {
+                // order creation unsuccessful!
+                setLoading(false);
+                setError(JSON.stringify(order_response_data));
+                return;
+            }
+            setOrderId(order_response_data.id);
+
+            // get payment method options // order creation successful
+            const new_payment_method_options_url = payment_method_options_url + `?country=IND&saved_payment_method=false&order_id=${order_response_data.id}`;
```

Comment:
`country` must not be hardcoded. Lets put it out as a variable and refer here. Like [here](https://github.com/inaitech/inai-web-sample-integration/blob/main/dropin-checkout/client/vanilla/pages/checkout.html#L18)